mission-control-client/src/styles/components/admin-styles/admin-dashboard-content.scss

```diff
@@ -11,18 +11,14 @@
     background: $secondary;
     color: white;
     border-radius: 5px;
-    padding: 2rem;
+    padding: 3rem;
 
     .dashboard-banner-head {
+      font-size: 30px;
```

Comment:
Should this be a rem?

Reply:
Agreed let's try to use rems when we can

Reply:
changed